### DISCUSSION TITLE
Incorrect docstring on HTTPStatusError

### DISCUSSION THREAD

**Discussion by nhoad** [encode/httpx#1932] [Potential Issue]
https://github.com/encode/httpx/blob/master/httpx/_exceptions.py#L226

>The response had an error HTTP status of 4xx or 5xx.

It should say something like "The response had a non-2xx HTTP status."

**Comment by lovelydinosaur** [comment]
Absolutely correct, yup. Thanks!
Anyone want to issue a tweak pull request for this?
